driver/razeraccessory_driver.c

```diff
@@ -631,6 +649,35 @@ static ssize_t razer_attr_write_matrix_effect_static(struct device *dev, struct
         request = razer_chroma_extended_matrix_effect_static(VARSTORE, ZERO_LED, (struct razer_rgb*) & buf[0]);
         request.transaction_id.id = 0x1F;
         break;
+    case USB_DEVICE_ID_RAZER_MOUSE_DOCK_PRO:
+        // Must be in normal mode for hardware effects
+        razer_set_device_mode(device, 0x00, 0x00);
+        /**
+            * Mode switcher required after setting static color effect once and before setting a second time.
+            * Similar to Naga Trinity?
+            *
+            * If the color is not set twice with the mode switch in-between, each subsequent
+            * setting of the static effect actually sets the previous color...
+            */
+        request = razer_chroma_extended_matrix_effect_static(VARSTORE, ZERO_LED, (struct razer_rgb*) & buf[0]);
+        request.transaction_id.id = 0xFF;
+
+        razer_send_payload(device, &request, &response);
+
+        request = get_razer_report(0x0f, 0x02, 0x06);
+        request.arguments[0] = 0x00;
+        request.arguments[1] = 0x00;
+        request.arguments[2] = 0x08;
+        request.arguments[3] = 0x00;
+        request.arguments[4] = 0x00;
+        request.arguments[5] = 0x00;
```

Comment:
I think you can remove the assignments to 0x00? So only keep arguments[2]. Also any clue what this call does?

Reply:
the arguments are already initialized to zero, so if you mean that with "for safety", then you can really remove that.